packages/paste-core/components/modal/__tests__/index.spec.tsx

```diff
@@ -156,7 +156,7 @@ describe('Modal', () => {
     expect(document.activeElement).toEqual(getByTestId('modal-header').querySelector('button'));
   });
 
-  it('should focus on the element provided as the initialFocus element in the modal', () => {
+  it.skip('should focus on the element provided as the initialFocus element in the modal', () => {
```

Comment:
This test causes a blow up for some reason now. totally can't figure out what is up with it.

Reply:
will you investigate that in another PR?

Reply:
Whoa, actual bug, terrible message. Good catch Jest

Reply:
Fixed